spec/ruby/core/time/new_spec.rb

```diff
@@ -456,7 +456,7 @@ def zone.local_to_utc(t)
         Time.new(2000, 1, 1, in: "+05:00").should == Time.new(2000, 1, 1, 0, 0, 0, "+05:00")
         Time.new(2000, 1, in: "+05:00").should == Time.new(2000, 1, 1, 0, 0, 0, "+05:00")
         Time.new(2000, in: "+05:00").should == Time.new(2000, 1, 1, 0, 0, 0, "+05:00")
-        Time.new(in: "+05:00").round(0).should == Time.now.getlocal("+05:00").round(0)
+        Time.new(in: "+05:00").tap { |time| time.should == time.getlocal("+05:00") }
```

Comment:
Not sure whether it's a correct change.

`time == time.getlocal(utc_offset)` will always be `true` for any Time instance.

Reply:
I suppose it might be
```ruby
Time.new(in: "+05:00").tap { |t| t.should == Time.new(t.year, t.month, t.day, t.hour, t.min, t.sec, "+05:00").
```

And probably with `.round` as well to reset milliseconds.

Reply:
Ah, right, I thought Time#== would care about the zone, but indeed it does not.
Maybe we could just use the original approach, but then use be_close like `time1.should be_close(time2, TIME_TOLERANCE)`.
Could you try that and push to my PR on BitBucket?

Reply:
Yes, sure